client/actions/projects.js

```diff
@@ -90,6 +91,17 @@ export function updateProject(project) {
   };
 }
 
+export function saveReusableSteps(reusableSteps) {
+  return (dispatch, getState) => {
+    const state = getState();
+    const newState = cloneDeep(state.project);
+    const updatedReusableSteps = keyBy(reusableSteps, 'id');
+    newState.reusableSteps = {...newState.reusableSteps, ...updatedReusableSteps};
+    dispatch(updateProject(newState));
+    return debouncedSyncConditions(dispatch, getState);
```

Comment:
Using `debouncedSyncConditions` and not `debouncedSyncProject` here feels _very_ odd to me. Is there a reason to use the conditions sync and not the normal project sync?

I'm surprised this even works given the conditions sync should make requests to an endpoint that is only accessible to ASRU users.

Reply:
Good spot, I will fix this locally and check everything is working as expected.

It looks as though the conditions sync is only syncing if there is data there so wouldn't have called the endpoint.

Reply:
I suspect it's not doing _anything_ then, and it's being correctly synced after the fact by another call to the sync function.

Reply:
This has been updated to use the `debouncedSyncProject`